drivers/regulator/regulator_gpio.c

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2019-2020 Peter Bigot Consulting, LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT regulator_gpio
+
+#include <drivers/regulator.h>
+#include <drivers/gpio.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(regulator_gpio, CONFIG_REGULATOR_LOG_LEVEL);
+
+#define OPTION_BOOT_ALWAYS_ON_POS 0
+#define OPTION_BOOT_ALWAYS_ON BIT(OPTION_BOOT_ALWAYS_ON_POS)
+#define OPTION_BOOT_ON_POS 0
+#define OPTION_BOOT_ON BIT(OPTION_BOOT_ON_POS)
```

Comment:
Should these be on different bit positions? Or that "always on" needs to have 2 bits (one for itself, and the other overlapping "boot_on")?